composer.json

```diff
@@ -21,16 +21,16 @@
         "phpcr/phpcr-implementation": "^2.1.0",
         "phpcr/phpcr-utils": "^1.3.0",
         "doctrine/instantiator": "^1.0.1",
-        "symfony/console": "^3.3"
+        "symfony/console": "^3.3 || ^4.0"
     },
     "require-dev": {
-        "symfony/yaml": "^3.3",
-        "symfony/phpunit-bridge": "^3.2",
-        "liip/rmt": "~1.2",
+        "symfony/yaml": "^3.3 || ^4.0",
+        "symfony/phpunit-bridge": "^3.2 || ^4.0",
+        "liip/rmt": "1.2.*",
```

Comment:
sorry, i seem to have been confused yesterday. no idea where i found the 2. whatever... liip/RMT had no branch-alias, so the thing did not work. i now added that. can you please change this to `1.3.*@dev`? this time it should work, packagist sees the 1.3 version.